src/engine/sequencer.js

```diff
@@ -225,7 +225,9 @@ class Sequencer {
             // Look for warp-mode flag on definition, and set the thread
             // to warp-mode if needed.
             const definitionBlock = thread.target.blocks.getBlock(definition);
-            const doWarp = definitionBlock.mutation.warp;
+            const innerBlock = thread.target.blocks.getBlock(
+                definitionBlock.inputs.custom_block.block);
```

Comment:
This is assuming that a whole chain of things exists (include a block connected to the input on the definition block).  What happens if it's missing?

Reply:
Yeah, fixed in follow up commit e8c5bbb